build.ts

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env node
+import * as child_process from "child_process";
+import * as fs from "fs";
+import { isDeepStrictEqual } from "util";
```

Comment:
Hi @tsmock, have you possibly forgotten to commit the isDeepStrictEqual from util?

Reply:
No. That should be in NodeJS 9+.

Reply:
Hi, sorry for the noise, my bad. Wasn't aware of such helpful utility functions. And tried to run the script using https://bun.sh/ which does not include isDeepStrictEqual.

Reply:
No worries. I don't mind answering questions (and TIL that bun.sh exists). Anyway, it looks like a good chunk of the `util` functions are [not supported](https://github.com/oven-sh/bun/discussions/852) by bun.sh.